<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocketAppQmake</class>
 <widget class="QWidget" name="SocketAppQmake">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SocketApp</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_protocol">
         <property name="text">
          <string>Protocol</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cb_protocol"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_sendfile">
         <property name="text">
          <string>Send File Path</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_sendfile">
         <property name="text">
          <string>/home/qasim/Desktop/Rampup</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_receivefile">
         <property name="text">
          <string>Receive File Path</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="le_receivepath">
         <property name="text">
          <string>/home/qasim/Desktop/Receive</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lbl_ip">
         <property name="text">
          <string>IP Address</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="le_ip">
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lbl_port">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="le_port">
         <property name="text">
          <string>1234</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="pb_send">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPlainTextEdit" name="te_status"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
